src/test/java/guitests/guihandles/MainGuiHandle.java

```diff
@@ -49,4 +59,25 @@ public void use_APP_RESIZE_HOTKEY() {
         guiRobot.sleep(1000);
     }
 
+    public FxRobot sleepForGracePeriod() {
+        return guiRobot.sleep((ModelManager.GRACE_PERIOD_DURATION + 1), TimeUnit.SECONDS);
+    }
+
+    public FxRobot sleep(long duration, TimeUnit timeunit) {
+        return guiRobot.sleep(duration, timeunit);
+    }
```

Comment:
I don't want all handle to be able to call sleep. Otherwise, we will handle personList.sleep(), statusBar.sleep(), etc. all over the testing codes.